docs/reference_dataset/template_repository/shared-config.yml

```diff
@@ -9,7 +9,8 @@ default:
     name: bosh-openstack-kvm-ubuntu-trusty-go_agent  # Default: bosh-openstack-kvm-ubuntu-trusty-go_agent
   concourse:
     parallel_execution_limit: 5 # Default: -1, ie unlimited
-
+    serial_group_naming_strategy: SerialGroupNameRoundRobinStrategy #Default: SerialGroupNameRoundRobinStrategy
```

Comment:
Would be useful to document to COA users criteria to use the naming strategy. Seems not obvious as of now

Reply:
I've added some explainations